index.html

```diff
@@ -59,6 +59,12 @@ <h2>WELCOME</h2>
 			</div>
 		</div>
 
+		<footer>
+			<p1>Authors: Deepak Kumar, Drake Prebyl, James Kraijcek, Rafael Alaras, Rodridgo Figueroa Justiniano.</br>
```

Comment:
<p1> is not a real HTML tag and therefore doesn't do anything, so these should be removed.